Makefile

```diff
@@ -12,13 +12,13 @@ REPORTS = $(ROOT_DIR)/reports
 SKIPPER_PARAMS ?= -i
 
 # assisted-service
-SERVICE_BRANCH := $(or $(SERVICE_BRANCH), "master")
+SERVICE_BRANCH := $(or $(SERVICE_BRANCH), "origin/master")
 SERVICE_REPO := $(or $(SERVICE_REPO), "https://github.com/openshift/assisted-service")
 SERVICE := $(or $(SERVICE), quay.io/ocpmetal/assisted-service:latest)
 SERVICE_NAME := $(or $(SERVICE_NAME),assisted-service)
 
 # assisted-installer
-INSTALLER_BRANCH := $(or $(INSTALLER_BRANCH), "master")
+INSTALLER_BRANCH := $(or $(INSTALLER_BRANCH), "origin/master")
```

Comment:
if there is example in the doc, we should change it to have origin in it

Reply:
right